spacy/lang/en/tokenizer_exceptions.py

```diff
@@ -396,6 +396,14 @@
         {ORTH: "Let", LEMMA: "let", NORM: "let"},
         {ORTH: "'s", LEMMA: PRON_LEMMA, NORM: "us"},
     ],
+    "c'mon": [
+        {ORTH: "c'm", NORM: "come", LEMMA: "come"},
+        {ORTH: "on"}
+    ],
+    "C'mon": [
+        {ORTH: "C'm", NORM: "come", LEMMA: "come"},
+        {ORTH: "on"}
+    ]
```

Comment:
@honnibal : do we want to keep this exception in?

Reply:
I'll go ahead and merge the PR with it so it's in sync – but we should revisit this when we update the exceptions for v3 (we'll also need to touch this part anyways when we kill the `PRON_LEMMA`).